integrationtests/kubernetes_with_k8s_module_test.go

```diff
@@ -100,18 +101,18 @@ func TestKubernetesDeployment(t *testing.T) {
 	}
 
 	terraform.Apply(t, terraformOptions)
-	assertDeploymentStatus(t, terraformOptions)
+	assertDeploymentStatus(t, terraformOptions, sk)
 	terraform.Destroy(t, terraformOptions)
 }
 
-func assertDeploymentStatus(t *testing.T, terraformOptions *terraform.Options) {
+func assertDeploymentStatus(t *testing.T, terraformOptions *terraform.Options, sk string) {
 	t.Helper()
 
 	masterYggIP := terraform.Output(t, terraformOptions, "master_yggip")
 	assert.NotEmpty(t, masterYggIP)
 
 	time.Sleep(5 * time.Second)
-	res, err := tests.RemoteRun("root", masterYggIP, "kubectl get node")
+	res, err := tests.RemoteRun("root", masterYggIP, "kubectl get node", sk)
 	res = strings.Trim(res, "\n")
 	assert.Empty(t, err)
 
```

Comment:
in the code after you can use strings.Count to count the word `ready` instead of spliting and looping